### DISCUSSION TITLE
Can it export all notes in a collection and all its subcollections while keeping the folder structure?

### DISCUSSION THREAD

**Discussion by Zheng392** [windingwind/zotero-better-notes#999] [General]
I would like to export all notes in a collection and all its subcollections. Also, I want to keep the structure of these collections and notes. It will be converted into a folder with subfolders and markdown notes. How can I do this? Or would it be possible to add this feature?

**Comment by windingwind** [comment]
You may select all in a collection and export via the context menu. However, it's not possible to automatically export multiple collections' notes, since a note can have multiple parent collections, thus it would not be able to know which folder to place the exported file.

**Comment by Zheng392** [comment]
One idea could be to export the same note in different collections into different folders. Then Betternotes could record the mapping and sync automatically. If the note is changed in Zotero, all notes in different folders will be changed at the same time. If the note is changed in a folder outside Zotero, the note in Zotero will be updated and then the same note in different folders will be updated too.

**Reply by windingwind** [reply]
Unfortunately, that's not possible, since these markdown files can conflict with each other, making the syncing process to be much too complicated.
